### PR TITLE
Issue 846 better syntax for annotated paragraph

### PR DESCRIPTION
**Pull request by moufort**
Issue #846 
I change the !! syntax to >[!

### CODE REVIEW

File: src/Microdown/MicAbstractAnnotatedBlock.class.st [hunk 1]
```diff
@@ -18,14 +18,18 @@ MicAbstractAnnotatedBlock >> addLineAndReturnNextNode: line [
...
-			| indexOfFirstSpace |
-			indexOfFirstSpace := line indexOf: Character space.
-			label := indexOfFirstSpace = 0
-				ifTrue: [ '' ]
-				ifFalse: [line copyFrom: self lineMarkup size + 1 to: indexOfFirstSpace - 1.].
-			body := (line copyFrom: indexOfFirstSpace + 1 to: line size) trim.]
-		ifNotNil: [ body = '' ifTrue: [ body := body , line ]
-			ifFalse: [ body := body , String cr , line ]]
+			| indexOfFirstClosingBracket |
+			indexOfFirstClosingBracket := line indexOf: $].
```
Comment (Ducasse) [inline src/Microdown/MicAbstractAnnotatedBlock.class.st:22]: I wonder what is the impact if the line does not contain ]

File: src/Microdown-Tests/MicAnnotatedParagraphTest.class.st [hunk 2]
```diff
@@ -25,12 +26,39 @@ MicAnnotatedParagraphTest >> testBasicAnnotetedParagraph [
 		equals: 'This is an important information'
 ]
 
+{ #category : 'tests' }
+MicAnnotatedParagraphTest >> testBasicAnnotetedParagraphIgnoreExtraTextInFirstLine [
```
Comment (Ducasse) [inline src/Microdown-Tests/MicAnnotatedParagraphTest.class.st:30]: Super nice test :)

File: src/Microdown/MicAbstractAnnotatedBlock.class.st [hunk 3]
```diff
@@ -41,6 +45,22 @@ MicAbstractAnnotatedBlock >> canConsumeLine: line [
 	^ line isNotEmpty 
 ]
 
+{ #category : 'as yet unclassified' }
```
Comment (Ducasse) [inline src/Microdown/MicAbstractAnnotatedBlock.class.st]: could categorize the method.

**Comment by Ducasse** [comment]
The following tests are not passing

```
✗ #testAnnotatedBlockTextElements (1ms)
 ✗ #testAnnotatedBlockWithListParserLogic (1ms)
 ✗ #testAnnotatedBlock (3ms)
 ✗ #testAnnotatedBlockOnMultipleLinesParserLogic (2ms)
```

File: src/Microdown/MicAbstractAnnotatedBlock.class.st (hunk 1 above)
Comment (moufort) [reply src/Microdown/MicAbstractAnnotatedBlock.class.st:22]: After getting the index of the first ], I test if he egal to 0.
If true, I removed the AnnotatedParagraphe to MicRootBlock, and I replace by a MicBlockQuoteBlock

indexOfFirstClosingBracket = 0
				ifTrue: [ parent children removeLast. ^ self createMicBlockQuoteBlock: line ]

**Pull request merged**